pawk.py

```diff
@@ -132,10 +132,7 @@ def _parse_command(arg):
 class Context(dict):
     def apply(self, numz, line, headers=None):
         l = line.rstrip()
-        if self.strict_field_splitting:
-            f = [w for w in l.split(self.delim)]
-        else:
-            f = [w for w in l.split(self.delim) if w]
+        f = [w for w in l.split(self.delim)]
```

Comment:
I think this can just be `f = l.split(self.delim)` now!

Reply:
Ah. Of course so!